csv/src/test/java/com/fasterxml/jackson/dataformat/csv/FeaturesTest.java

```diff
@@ -26,4 +29,13 @@ public void testFactoryFeatures() throws Exception
         assertTrue(g.canUseSchema(CsvSchema.emptySchema()));
         g.close();
     }
+
+    public void testFactoryFastFeatures() throws Exception
+    {
+        CsvFactory f = new CsvFactory();
```

Comment:
Maybe use builder instead, to make the test work for 3.0 too?

Reply:
I've added an extra test for this